app/services/imports/sales_logs_field_import_service.rb

```diff
@@ -12,6 +14,8 @@ def update_field(field, folder)
   private
 
     def update_creation_method(xml_doc)
+      return if meta_field_value(xml_doc, "form-name").include?("Lettings")
```

Comment:
unrelated but I think this should be here (same for the change to lettings_logs_field_import_service)

Reply:
Hmm, I'd say maybe change it to this: `return unless meta_field_value(xml_doc, "form-name").include?("Sales")`
Cause I seem to recall that not all the lettings form names include `Lettings`, that's why we're only checking for sales or not sales everywhere else we do this

Reply:
ahh nice ok, good point, thanks